src/deepsparse/yolact/utils/annotate.py

```diff
@@ -0,0 +1,194 @@
+from typing import Optional, Tuple
+
+import numpy
+import numpy as np
+
+import cv2
+from deepsparse.yolact.schemas import YOLACTOutputSchema
+from deepsparse.yolo.utils.utils import _get_color, _plot_fps
+
+
+def annotate_image(
+    image: numpy.ndarray,
+    prediction: YOLACTOutputSchema,
+    images_per_sec: Optional[float] = None,
+    score_threshold: float = 0.35,
+) -> numpy.ndarray:
+    """
+    Annotate and return the img with the prediction data
+
+    The function will:
+    - draw bounding boxes on predictions of a model
+    - annotate every prediction with its proper label
+    - draw segmentation mask on the image
+
+    Note: in private functions:
+     - `_put_mask()`
+     - `_put_annotation_text()`
+     - `_put_bounding_box()`
+     - `_get_text_size()`
+
+    there are some hard-coded values that parameterize the layout of the annotations.
+    You may need to adjust the parameters to improve the aesthetics of your annotations.
+
+    :param image: original image to annotate (no pre-processing needed)
+    :param prediction: predictions returned by the inference pipeline
+    :param images_per_sec: optional fps value to annotate the left corner
+        of the image (video) with
+    :param score_threshold: minimum score a detection should have to be annotated
+        on the image. Default is 0.35
+    :return: the original image annotated with the given bounding boxes
+        (if predictions are not None)
+    """
+
+    masks = prediction.masks[0]
+    boxes = prediction.boxes[0]
+    classes = prediction.classes[0]
+    scores = prediction.scores[0]
+
+    if any(x[0] is None for x in [masks, boxes, classes, scores]):
+        # no detections found
+        return image
+
+    image_res = copy.copy(image)
+
+    masks, boxes = _resize_to_fit_img(image, masks, boxes)
+
+    for box, mask, class_, score in zip(boxes, masks, classes, scores):
+        if score > score_threshold:
+            colour = _get_color(class_)
+            left, top, _, _ = box
+            image_res = _put_mask(image=image_res, mask=mask, colour=colour)
+            image_res = _put_bounding_box(image=image_res, box=box, colour=colour)
+
+            annotation_text = f"{class_}: {score:.0%}"
+            text_width, text_height = _get_text_size(annotation_text)
+            image_res = _put_annotation_text(
+                image=image_res,
+                annotation_text=annotation_text,
+                left=left,
+                top=top,
+                colour=colour,
+                text_width=text_width,
+                text_height=text_height,
+            )
+
+    if images_per_sec is not None:
+        image_res = _plot_fps(
+            img_res=image_res,
+            images_per_sec=images_per_sec,
+            x=20,
+            y=30,
+            font_scale=0.9,
+            thickness=2,
+        )
+
+    return image_res
+
+
+def _put_mask(
+    image: numpy.ndarray, mask: numpy.ndarray, colour: Tuple[int, int, int]
```

Comment:
very bad nit, can be absolutely ignored, but we generally use american english in our code base

```suggestion
    image: numpy.ndarray, mask: numpy.ndarray, color: Tuple[int, int, int]
```
Kindly update other usages accordingly if applied